pkg/config/templates/rancherd-14-logging-crd.yaml

```diff
@@ -0,0 +1,18 @@
+resources:
+- apiVersion: management.cattle.io/v3
+  kind: ManagedChart
+  metadata:
+    name: rancher-logging-crd
+    namespace: fleet-local
+  spec:
+    chart: rancher-logging-crd
+    releaseName: rancher-logging-crd
+    version: {{ .LoggingChartVersion }}
+    defaultNamespace: cattle-monitoring-system
```

Comment:
defaultNamespace: cattle-monitoring-system -> cattle-logging-system

Reply:
changed to `cattle-logging-system`